Makefile

```diff
@@ -201,21 +202,29 @@ format:
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
+PROTO_BUILDER_IMAGE=tendermintdev/sdk-proto-gen@sha256:372dce7be2f465123e26459973ca798fc489ff2c75aeecd814c0ca8ced24faca
```

Comment:
Cool, they provide these standard docker images now

Reply:
ja, I dropped our image for this.